EI patterns/event_message/main.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/http;
+import ballerina/mime;
+import ballerina/url;
+
+listener http:Listener httpListener = new (port = 8080);
+
+service /api/v1 on httpListener {
+    resource function post incidents(IncidentRequest req) returns error? {
+        string body = string `Incident ${req.incident.description} reported: ${req.incident.date} at ${req.incident.time}.`;
+        http:Request twilioReq = new http:Request();
+        string payload = "From=" + check url:encode("+15005550006", "utf-8") +
+"&To=" + check url:encode(req.phoneNo, "utf-8") +
+"&Body=" + check url:encode(body, "utf-8");
```

Comment:
It seems this is an issue in Formatter. @chiranSachintha please check.

Reply:
This is formatting issue will create